proto/minder/v1/minder.proto

```diff
@@ -31,12 +31,49 @@ enum ObjectOwner {
     reserved 1; // deprecated OBJECT_OWNER_ORGANIZATION
 }
 
+enum Relation {
+    RELATION_UNSPECIFIED = 0;
```

Comment:
Do we need a dedicated relation to mark that an API endpoint doesn't need to go through authorization? e.g. the User endpoints which only apply to the current user.

Reply:
I was imagining authorization to depend on the relation and the `ObjectOwner`. In the case of user endpoints, the object owner is the user, so we wouldn't bother looking that the relation.

Reply:
oh! I was thinking of ditching the `ObjectOwner`, `ProjectOwner` etc. pieces altogether.

Reply:
We could. I kind of like having the relations closely tied to the openfga model, and having a separate way of deciding if we should even call openfga.

Reply:
The `Object/Project/Org` ownership, while something we have now, seems like could it could be easy to forget. What if we have a separate mechanism and deprecate the other way?

Reply:
That works, I'm not attached to the `ObjectOwner` enum.
One option could be adding an enum for a target resource type.
```
enum TargetResource {
    TARGET_RESOURCE_PROJECT = 0;
    TARGET_RESOURCE_NONE = 1;
    TARGET_RESOURCE_USER = 2;
}
```
Then `TARGET_RESOURCE_NONE` could also replace our current usage of `anonymous`.

Do you have any other ideas?

Reply:
I like your proposal. I think we should go for that. It feels a lot more intuitive.